app/views/components/widget/_widget.css.erb

```diff
@@ -1033,11 +972,6 @@
   text-decoration:none;
 }
 
-/* Override */
-.fba-modal-dialog .usa-form {
-  max-width: inherit;
-}
```

Comment:
This rule was making the input fields too wide. They looked like this:

<img width="792" alt="Screenshot 2024-08-05 at 3 17 02 PM" src="https://github.com/user-attachments/assets/235ac314-ea52-4b1a-8f84-f00027efa762">